src/pages/Homepage.js

```diff
@@ -53,6 +54,10 @@ const Homepage = (props) => {
       console.log(error)
     }
   }
+
+  function removeAll (list) {
+    list.forEach(async (item) => item.completed ? await remove(item._id) : '')
```

Comment:
the `forEach` loop is passive, and is not awaiting the promises from the `remove` calls. Instead, consider making `removeAll` `async`. and doing something like:

```js
await Promise.all(
  // .map() will return an Array of Promises. We can wait for all of them to resolve with Promise.all
  list.map(async (item) => item.completed ? await remove(item._id) : '')
)
```

With that, `removeAll` will return a `Promise` that resolves when all of the remove calls resolve.

Reply:
ok. No problem.